templates/.github/workflows/go-check.yml

```diff
@@ -16,6 +18,12 @@ jobs:
       - name: Run repo-specific setup
         uses: ./.github/actions/go-check-setup
         if: hashFiles('./.github/actions/go-check-setup') != ''
+      - name: Read config
+        if: hashFiles('./.github/workflows/go-check-config.json') != ''
```

Comment:
So, we _could_ just deploy a `test.env` file when we deploy CI.

Reply:
That would let us drop jq.

Reply:
Would that be easier? For a simple configuration like the one introduced here maybe, but in the future we might have more complex options.

Reply:
I'd like to avoid having to put JSON into the `test.env` file at some point.

Reply:
I assume it would come down to running the logic on deploy, versus afterwards. But it's not critical either way.